shared/constants/chat2/meta.tsx

```diff
@@ -431,6 +433,15 @@ export const getCommands = (state: TypedState, id: Types.ConversationIDKey) => {
   }
 }
 
+export const getBotCommands = (state: TypedState, id: Types.ConversationIDKey) => {
+  const {botCommands} = getMeta(state, id)
+  if (botCommands.typ === RPCChatTypes.ConversationCommandGroupsTyp.custom && botCommands.custom) {
+    return botCommands.custom.commands || []
```

Comment:
should hoist this empty array up else your containers will thrash

Reply:
Is there like a global empty array? We will never hit this case anyway, this is just to appease TS, the service will always be providing a value here.

Reply:
no. this is why traditionally we use immutable for these types of datastructures but you're using an array and raw rpc stuff